requirements.txt

```
@@ -1,7 +1,7 @@
 black==22.3.0
 flake8==5.0.4
 flake8-tidy-imports==4.6.0
-graviton@git+https://github.com/algorand/graviton@v0.5.0
+graviton@git+https://github.com/algorand/graviton@19959cca5a4ca7aaf4736a21c7c23f2083a54233
```

```suggestion
graviton@git+https://github.com/algorand/graviton@v0.6.0
```
Change after [graviton #36](https://github.com/algorand/graviton/pull/36/files) merged and the new tag is added